core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.html

```diff
@@ -503,10 +503,10 @@ <h3>Feedback/Suggestion Email Preferences</h3>
             </div>
           </div>
         </div>
-        <span class="form-text secondary-info-text" ng-if="!$ctrl.UserEmailPreferencesService.areFeedbackNotificationsMuted()">
+        <span class="form-text secondary-info-text" ng-if="$ctrl.UserEmailPreferencesService.areFeedbackNotificationsMuted()">
```

Comment:
hey @kevintab95, correct me if I'm wrong but isn't this a bit wrong? The function in the ng-if directive and the text displayed in the div are the opposite of each other.

Reply:
Ah, yes, thanks for catching this @nithinrdy! The correct fix should be to swap the text.

Reply:
I was actually looking into the error and I believe the text is displayed just fine -- its the switch that is displayed incorrectly.

Reply:
(shows disabled when the emails are actually enabled, and vice-versa)